ci-operator/step-registry/baremetalds/packet/teardown/baremetalds-packet-teardown-commands.sh

```diff
@@ -6,6 +6,14 @@ set -o pipefail
 
 echo "************ baremetalds packet teardown command ************"
 
+set -x
+CIRFILE=$SHARED_DIR/cir
+if [ -e $CIRFILE ] ; then
```

Comment:
because I'm using "curl -f -o $CIRFILE" it will only be written out on success

wrapping it under a check against ${CLUSTER_PROFILE_DIR}/ofcir_url availability would be redundant as $CIRFILE can only exist with  ${CLUSTER_PROFILE_DIR}/ofcir_url,